src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp/MainPage.xaml

```diff
@@ -0,0 +1,14 @@
+<Page x:Class="MyExtensionsApp.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:MyExtensionsApp"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d"
+	Background="{ThemeResource BackgroundBrush}">
```

Comment:
BackgroundBrush doesn't exist so will cause WinUI to exception out with parse error. Uno seems more forgiving.
Use ApplicationPageBackgroundThemeBrush instead